### DISCUSSION TITLE
Custom error layout

### DISCUSSION THREAD

**Discussion by visini** [nuxt/nuxt#18318] [Questions] [answered]
In Nuxt v2 we could create `~/layouts/error.vue` to display errors in a customized way. This doesn't seem to work (yet?) in Nuxt v3...

Based on the following lines, it seems Nuxt v3 is currently displaying a static page from @nuxt/design – or am I missing something?

https://github.com/nuxt/framework/blob/a896c64b1232dba1d01874159a9953d5f79bccc4/packages/nitro/src/runtime/server/error.ts#L59-L61

**Comment by danielroe** [comment, marked as answer]
You can follow at https://github.com/nuxt/framework/discussions/559

**Reply by visini** [reply]
Thanks!
